src/Controller.php

```diff
@@ -34,7 +34,13 @@ private function getProvider($provider, $redirectUri = null)
     {
         $config = $this->getConfig($provider) + compact('redirectUri');
 
-        $class  = 'League\\OAuth2\\Client\\Provider\\' . ucfirst($provider);
+        if (isset($_ENV[strtoupper($provider) . '_NAMESPACE'])) {
+            $namespace = $_ENV[strtoupper($provider) . '_NAMESPACE'];
```

Comment:
Yeah I don't like it in the config, but I wasn't sure how else to get around it at the time. What if we just hard code some `use` lines at the top?